osmosis-1/osmosis.zone_assets.json

```diff
@@ -4065,6 +4065,15 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-06-25T12:00:00Z",
       "_comment": "N43 FanToken $N43"
+    },
+    {
+      "chain_name": "terra",
+      "base_denom": "ibc/46579C587A0B8CF8B0A1FF6B0EFA2082F11876578E47FC81A9CAAD31F424AF98",
+      "path": "transfer/channel-78488/cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+      "categories": ["social"],
```

Comment:
If this is a meme token, then I don't see how social applies. There is a category "meme".

Reply:
Corrected.